common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -85,6 +85,7 @@ remove_racial_traits_effect = {
 	if = { limit = { trait = creature_black_wyrm 			} remove_trait = creature_black_wyrm 			}
```

Comment:
You should add cultural building and retinue for their culture.
Here are the docs for building balance.
https://docs.google.com/spreadsheets/d/1wLs_i8YQRL4_BmxPBdoibJQTC0vJmu8RbidPy7cwGZY/edit#gid=2044635640
Retinues are balanced around having the same sum of bonuses.